jib-core/src/test/java/com/google/cloud/tools/jib/docker/DockerClientTest.java

```diff
@@ -16,6 +16,8 @@
 
 package com.google.cloud.tools.jib.docker;
 
+import static org.mockito.Mockito.when;
```

Comment:
nit: in our repo, we just do `Mockito.when(...)` without static import.

Reply:
addressed